changelog/unreleased/2895

```diff
@@ -0,0 +1,8 @@
+Enhancement: Add status indicator extension point
```

Comment:
=> "Add file list status indicator extension point" as this applies only to file list entries
(for those who are too lazy to read the description)